snp_motion_planning/src/planner_profiles.hpp

```diff
@@ -66,7 +66,9 @@ tesseract_planning::OMPLDefaultPlanProfile::Ptr createOMPLProfile()
 
   // Add as many planners as available threads so mulitple OMPL plans can happen in parallel
   auto profile = std::make_shared<tesseract_planning::OMPLDefaultPlanProfile>();
-  profile->planning_time = 20.0;
+  profile->planning_time = 5.0;
```

Comment:
5 seconds seems a little short to me. I know it's more convenient to fail fast when there's something wrong, but in my experience I've had to take longer and that's why I had it at 20.

Reply:
Was this for SNP applications or just OMPL in general? It seems like most of the SNP moves are straightforward enough for OMPL to find a solution in 5 seconds. I changed it because it's mostly just annoying to wait 20 seconds for OMPL to fail on "bad" plans that were never going to succeed in the first place (e.g., collision by default, etc.).

Reply:
It was for a scan n plan application, but I can't remember what demo exactly. I see that I changed it from 10 to 20 in my big "update to latest tesseract" PR. But you're right that for the vast majority of cases it plans quite fast

Reply:
Let's just make it a parameter with default set to 5 so you only have to set it if you want to change anything. Really I'd like pretty much all the profile configurations to be made up of parameters that are loaded at planning time.